tests/scripts/thread-cert/Cert_6_1_01_RouterAttach.py

```diff
@@ -51,19 +55,72 @@ def setUp(self):
         self.nodes[ED].add_whitelist(self.nodes[LEADER].get_addr64())
         self.nodes[ED].enable_whitelist()
 
+        self.sniffer = config.create_default_thread_sniffer(SNIFFER)
+        self.sniffer.start()
+
     def tearDown(self):
+        self.sniffer.stop()
+        del self.sniffer
+
         for node in list(self.nodes.values()):
             node.stop()
         del self.nodes
 
     def test(self):
         self.nodes[LEADER].start()
-        self.nodes[LEADER].set_state('leader')
+        time.sleep(5)
         self.assertEqual(self.nodes[LEADER].get_state(), 'leader')
 
         self.nodes[ED].start()
         time.sleep(5)
         self.assertEqual(self.nodes[ED].get_state(), 'child')
 
+        leader_messages = self.sniffer.get_messages_sent_by(LEADER)
+        ed_messages = self.sniffer.get_messages_sent_by(ED)
+
+        # 1 - leader
+        msg = leader_messages.next_mle_message(mle.CommandType.ADVERTISEMENT)
```

Comment:
sorry. I think it follows the style of the previous code. Good for code readability.